documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -0,0 +1,1503 @@
+= Debezium Connector for Db2 
+include::../_attributes.adoc[]
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+[NOTE]
+====
+Want to help us further hone and improve it? link:/docs/contribute/[Learn how].
+====
+
+toc::[]
+
+Debezium's Db2 Connector can monitor and record the row-level changes in the tables of a Db2 database.
+This connector was added in release Debezium 1.1.x.
+It is strongly inspired by the Debezium implementation of  SQL server, using a
+SQL based polling model of tables put into "capture mode". This uses the ASN libraries that come as a standard part of Db2 LUW and can be added (with an additional license) to Db2 zOS.
+
+The first time it connects to a Db2 database, it reads a consistent snapshot of all of the tables that are whitelisted (or not blacklisted depending on the mode of operation).
+Note that by default all tables in the database are snapshoted, NOT just those that are in capture mode.
+
+When that snapshot is complete, the connector continuously streams the changes that were committed to the Db2 database for all whitelisted tables in capture mode. This generates corresponding insert, update and delete events.
+All of the events for each table are recorded in a separate Kafka topic, where they can be easily consumed by applications and services.
+
+[[overview]]
+== Overview
+
+The functionality of the connector is based upon the ASN Capture/Apply agents 
+that enable SQL Replication within DB2:
+https://www.ibm.com/support/pages/q-replication-and-sql-replication-product-documentation-pdf-format-version-101-linux-unix-and-windows
```

Comment:
Should be a link with proper text, e.g. in the "ASN Capture/Apply agents" part in the sentence.

Reply:
Done